portal-wire-protocol.md

```diff
@@ -213,6 +213,22 @@ selector = 0x02
 ssz-type = List[ByteList[2048], 32]
 ```
 
+
+##### Content encoding for uTP transfer
```

Comment:
```suggestion
##### Length prefixing for utp transfers
```

Reply:
The acronym for uTP is uTP as in `uTorrent Transport Protocol`

Reply:
updated.